packages/api/src/routes/health.ts

```
@@ -0,0 +1,165 @@
+import { Router, type Request, type Response } from 'express';
```

ESLint

[error] 'Request' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)